### PR TITLE
solution

### PR DESCRIPTION
**Pull request by igor-kashchenko**
[DEMO LINK](https://igor-kashchenko.github.io/react_people-table-basics/)

### CODE REVIEW

File: src/components/PeopleTable/PeopleTable.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              <td>
+                <PersonLink person={person} />
+              </td>
+
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+
+              <td>
+                {!mother ? (motherName || '-') : <PersonLink person={mother} />}
```
Comment (anna-inozemets) [inline src/components/PeopleTable/PeopleTable.tsx:58]: (motherName || '-') - maybe, it is better to create a variable for this statement

File: src/components/PeopleTable/PeopleTable.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            motherName,
+            slug,
+          } = person;
+
+          const isSelected = slug === personSlug;
+          const mother = people.find(mom => mom.name === motherName);
+          const father = people.find(dad => dad.name === fatherName);
+
+          return (
+            <tr
```
Comment (anna-inozemets) [inline src/components/PeopleTable/PeopleTable.tsx:44]: I think, we can create a separate component for this part, smth like "PersonItem"

**Review by anna-inozemets — APPROVED** [review]
Clear and good code!)

File: src/components/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { PageNavLink } from '../PageNavLink';
+
+export const NavBar: React.FC = () => {
+  return (
```
Comment (denys-cheporniuk) [inline src/components/NavBar/NavBar.tsx:5]: you can avoid return here

File: src/components/PageNavLink/PageNavLink.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => {
+  return (
```
Comment (denys-cheporniuk) [inline src/components/PageNavLink/PageNavLink.tsx:11]: the same with return

File: src/components/PeopleTable/PeopleTable.tsx (hunk 2 above)
Comment (denys-cheporniuk) [reply src/components/PeopleTable/PeopleTable.tsx:44]: agree, PersonItem looks good here

File: src/components/PeopleTable/PeopleTable.tsx (hunk 1 above)
Comment (denys-cheporniuk) [inline src/components/PeopleTable/PeopleTable.tsx:58]: ternary operator should return one value depending on condition, don't make another condition in the ternary operator

File: src/components/PeopleTable/PeopleTable.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+              <td>{sex}</td>
+              <td>{born}</td>
+              <td>{died}</td>
+
+              <td>
+                {!mother ? (motherName || '-') : <PersonLink person={mother} />}
+              </td>
+
+              <td>
+                {!father ? (fatherName || '-') : <PersonLink person={father} />}
```
Comment (denys-cheporniuk) [inline src/components/PeopleTable/PeopleTable.tsx:62]: the same here with ternary operator

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**